src/SFA.DAS.EmployerIncentives.Api/Models/UpdateApplicationRequest.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace SFA.DAS.EmployerIncentives.Api.Models
+{
+    public class UpdateApplicationRequest
+    {
+        public Guid ApplicationId { get; set; }
+        public long AccountId { get; set; }
+        public long AccountLegalEntityId { get; set; }
```

Comment:
I thought we weren't sending in the AccountLegalEntity as it already exists on the application and can not be changed

Reply:
good spot!